resume.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <!-- <link href="./main.css" rel="stylesheet" /> -->
+    <link href="css\main.css" rel="stylesheet" />
+    <title>Responsive Resume Page</title>
+  </head>
+
+  <body>
+    <!-- HEADER -->
+    <header>
+      <nav>
+        <div class="box"></div>
+        <h1 class="title_top">
+          Maya Nelson </h1>
+  
+        <div class="ham_menu">
+          <div class="wrapper-menu">
+            <div class="line-menu half start"></div>
+            <div class="line-menu"></div>
+            <div class="line-menu half end"></div>
+          </div>
+        </div>
+  
+        <p>PROJECT MANAGER</p>
+        <ul id="menu" class="menu">
+          <li><a href="index.html">ABOUT ME</a></li>
+          <li><a href="./resume.html">RESUME</a></li>
+          <li><a href="#">PROJECTS</a></li>
+          <li><a href="#">CONTACT</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!-- MAIN -->
+    <main>
+      <!-- HEADING -->
+      <section class="title">
+        <div class="box2"></div>
+        <h2>Resume</h2>
+      </section>
+      <section class="headings">
+        <h2>Experience</h2>
+        <a
+          href="https://3f62d478-a066-43ee-98e6-3e48600ff893.filesusr.com/ugd/84770f_5db758741cef46fbb32e0125937f8de5.pdf"
+          target="_blank"
+        >
+          <button class="dwnload_btn">DOWNLOAD CV</button></a
+        >
+      </section>
+      <!-- ARTICLE START -->
+      <article class="art1">
```

Comment:
instead of using classes like `art1`, `art2`, `art3` etc I would try to create just one reusable class like you have done with `art`, the articles look like they are using similar styles so you could just use one class name for these